docs/hugo/content/blogs/release-notes-v2.16.0.md

```diff
@@ -0,0 +1,56 @@
+---
+title: "ASO v2.16 Release Notes"
+date: 2025-10-29
+description: "Release notes for Azure Service Operator v2.16.0"
+---
+
+We're excited to announce the release of Azure Service Operator v2.16.0! This release comes with a host of improvements and fixes, along with support being added for some oft-requested resources. 
+
+## 🎉 New and improved resource support
+
+### New resource support
+
+We've added support for three new resources: 
+
+- [Azure Compute Capacity Reservation Groups](https://github.com/Azure/azure-service-operator/pull/4980) allowing capacity reservation manifests to be declaratively included alongside VM manifests 
+- [Managing Azure quotas](https://github.com/Azure/azure-service-operator/pull/4979) for Kubernetes native management of Azure quotas
+- [NetworkWatchers/flowLogs](https://github.com/Azure/azure-service-operator/issues/4614) for management of Network Security Group Flow Logs.
+
+### Enhanced resource references
+
+We've enhanced existing resource references with additional functionality:
+
+- You can now specify Role Assignments using [well-known role defintion names](https://github.com/Azure/azure-service-operator/pull/4923) instead of GUIDs.
+- UserAssignedIdentity now supports [config map](https://github.com/Azure/azure-service-operator/pull/4940).
+- A system-managed identity can now be [specified in the app resources' identity references]((https://github.com/Azure/azure-service-operator/pull/4924)).
+- ResourceReferences now support [WellKnown names](https://github.com/Azure/azure-service-operator/pull/4922)
+
+## 🐛 Bug fixes
+
+- Fixed a [high-priority regression](https://github.com/Azure/azure-service-operator/pull/4966) from ASO v2.15.0 where resources failed with the `"/v1, Kind=Secret is not cached"` error during resource claims.
+- Kusto database creation should [no longer be attempted](https://github.com/Azure/azure-service-operator/pull/4976) while a cluster is stopped. 
+- `NetcfgSubnetRangeOutsideVnet` error is [now retryable](https://github.com/Azure/azure-service-operator/pull/4931) for VirtualNetworksSubnet to avoid the resources becoming stuck in certain scenarios.
+- Added a missing parameter for [default node pool](https://github.com/Azure/azure-service-operator/issues/4942) in Node Auto-Provisioning.
+- Fixed [select annotation changed predicate](https://github.com/Azure/azure-service-operator/pull/4967) from mistakenly classifying creates as containing annotation changes.
+- Fixed broken [ResourceReferenceProperties](https://github.com/Azure/azure-service-operator/pull/4925).
```

Comment:
You could move this up into the _Enhanced Resource References_ section, as it fixes things that were simple strings, but are now proper references. There are two affected resources (look at the `structure.txt` files in #4925 to find the detail).